Formula/qscintilla2.rb

```diff
@@ -52,18 +55,20 @@ def install
     # Add qscintilla2 features search path, since it is not installed in Qt keg's mkspecs/features/
     ENV["QMAKEFEATURES"] = "#{prefix}/data/mkspecs/features"
 
-    cd "Python" do
-      Language::Python.each_python(build) do |python, version|
-        (share/"sip").mkpath
-        system python, "configure.py", "-o", lib, "-n", include,
-                         "--apidir=#{prefix}/qsci",
-                         "--destdir=#{lib}/python#{version}/site-packages/PyQt4",
-                         "--qsci-sipdir=#{share}/sip",
-                         "--pyqt-sipdir=#{HOMEBREW_PREFIX}/share/sip",
-                         "--spec=#{spec}"
-        system "make"
-        system "make", "install"
-        system "make", "clean"
+    if (build.with? "python") || (build.with? "python3")
```

Comment:
It's a bit nicer to make that `if build.with?("python") || build.with?("python3")`.